modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java

```diff
@@ -456,7 +466,7 @@ else if (cpDefinition != null) {
 			product.getExternalReferenceCode(), commerceCatalog.getGroupId(),
 			LanguageUtils.getLocalizedMap(nameMap),
 			LanguageUtils.getLocalizedMap(shortDescriptionMap),
-			LanguageUtils.getLocalizedMap(descriptionMap), null,
+			LanguageUtils.getLocalizedMap(descriptionMap), urlTitleMap,
```

Comment:
Why don't you use LanguageUtils.getLocalizedMap method here?

Reply:
hey @andrea-ale-sbarra , if CPDefinition is null, the value would be:
`Map<Locale, String> urlTitleMap = LanguageUtils.getLocalizedMap(productUrlTitleMap);`

If CPDefinition is not null, the value would be:
`urlTitleMap = cpDefinition.getUrlTitleMap();`

Correct me if I am wrong. Isn't cpDefinition.getUrlTitleMap() already localized?

Reply:
Yes you're right, but I think the same is for the other maps for example short description. I think either you change also all the others or you use the same pattern for this. I'm fine with both solutions